src/main/plugins/org.dita.base/xsl/preprocess/topicpullImpl.xsl

```diff
@@ -61,11 +61,11 @@ mode="topicpull:figure-linktext" and mode="topicpull:table-linktext"
 
   <!-- Set the format for generated text for links to tables and figures.   -->
   <!-- Recognized values are 'NUMBER' (Table 5) and 'TITLE' (Table Caption) -->
-  <xsl:param name="TABLELINK">NUMBER</xsl:param>
-  <xsl:param name="FIGURELINK">NUMBER</xsl:param>
-  <xsl:param name="remove-broken-links" as="xs:string?"/>
+  <xsl:param name="TABLELINK" as="xs:string" select="'NUMBER'"/>  <!-- DITA-OT parameter args.tablelink.style: 'NUMBER' or 'TITLE' -->
```

Comment:
@chrispy-snps I marked this approved because I'm fine with the code comments - but would it be possible to move the code comment to the line above? I worry about a future code linting tool that will move the comment to the next line, and you end up with 
```
<xsl:param name="remove-broken-links" as="xs:string" select="'false'"/> 
<!-- DITA-OT parameter remove-broken-links: 'false' or 'true' -->
<xsl:param name="remove-broken-links" as="xs:string" select="'false'"/>
```

Reply:
Updated to request that change -- other than that LGTM